definitions/variable/macro-economy/employment.yaml

```diff
@@ -0,0 +1,5 @@
+- Employment|{Economic Sectors Level 2}:
+    description: Employment by the {Economic Sectors Level 2}
+    unit: Million full-time equivalent workers
+    tier: 1
```

Comment:
Does this really have to be a tier-1 variable?

Also, the top-level "Employment" variable was renamed to "Labour Force|Employed" in common-definitions. Consider revising here as well, or add a `note` attribute to cross-reference to the total-employment variable.